tasks/release.rb

```diff
@@ -44,13 +44,21 @@
       raise "Could not insert PRE in #{file}" unless $1
 
       File.open(file, 'w') { |f| f.write ruby }
+
+      if framework == "actioncable"
+        Dir.chdir("#{root}/actioncable") do
+          version = version.gsub(/\./).with_index { |s, i| i === 2 ? '-' : s }
```

Comment:
Take a look at https://rubygems.org/gems/rails/versions and https://github.com/rails/rails/releases to see our version style in history and at https://github.com/rails/rails/blob/v4.1.14.2/version.rb to see a 4-number version in practice.